test/accessibility/sitemap-1.spec.js

```diff
@@ -13,11 +13,23 @@ module.exports = {
       const mark = Math.ceil(urls.length / 4);
       const segment = urls.splice(0, mark);
 
+      // Whitelist of URLs on which to only run'section508' rules and not 'wcag2a'
+      // rules. Please provide an explanation for each exception.
+      const wcagSkip = [
```

Comment:
Actually, I think I'll move this whitelist into `sitemap-helpers.js` since it's not guaranteed which URL will be distributed to which of the `sitemap-*spec.js` files

@jbalboni  I'll ping you when this is ready for review